packages/components/src/internal/components/domainproperties/models.test.ts

```diff
@@ -1016,15 +1016,9 @@ describe('DomainField', () => {
 
         field = field.merge({ dataType: LOOKUP_TYPE }) as DomainField;
         expect(field.getDetailsArray().slice(0, 2).join('')).toBe('Updated. Current Folder > exp');
```

Comment:
Is there a different comparison we can do here? Are these no longer links?

Reply:
The tests are now executing in an environment that appears as SampleManager and this stuff is not rendered.